src/primitive_file_win.cc

```diff
@@ -354,8 +354,8 @@ int64 low_high_to_int64(DWORD high, DWORD low) {
 #define WINDOWS_TICKS_PER_SECOND 10000000
 #define SEC_TO_UNIX_EPOCH 11644473600LL
 
-Object* time_stamp(Process* process, FILETIME* time) {
-  int64 windows_ticks = low_high_to_int64(time->dwHighDateTime, time->dwLowDateTime);
+Object* time_stamp(Process* process, LARGE_INTEGER* time) {
```

Comment:
Why isn't this a `const LARGE_INTEGER&`? I think this is what you use below `file_time`.